### PR TITLE
Promo download

### PR DESCRIPTION
**Pull request by sertony**

### CODE REVIEW

File: src/ko.json [hunk 1]
```diff
@@ -976,6 +976,17 @@
...
     },
+    "promoDownload": {
+        "mobile": {
+            "appStore": "App Store",
+            "googlePlay": "Google Play",
+            "notice": "애플리케이션 다운로드"
+        },
+        "desktop": {
+            "download": "Download now for free",
+            "notice": "Try a new level of security with Surf for desktop devices"
```
Comment (sertony) [inline src/ko.json]: @kkkhesha can you translate, please?

File: src/ru.json [hunk 2]
```diff
@@ -965,6 +965,17 @@
...
     },
+    "promoDownload": {
+        "mobile": {
+            "appStore": "App Store",
+            "googlePlay": "Google Play",
+            "notice": "Установить приложение"
+        },
+        "desktop": {
+            "download": "Download now for free",
+            "notice": "Try a new level of security with Surf for desktop devices"
```
Comment (sertony) [inline src/ru.json]: @AnnStepanova can you translate, please?

File: src/ko.json [hunk 3]
```diff
@@ -976,6 +976,17 @@
...
         "rate": "서프 평가하기"
     },
+    "promoDownload": {
+        "mobile": {
+            "appStore": "App Store",
+            "googlePlay": "Google Play",
+            "notice": "애플리케이션 다운로드"
+        },
+        "desktop": {
+            "download": "Download now for free",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
            "download": "무료 다운로드하세요",
```

File: src/ko.json (hunk 1 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
            "notice": "데스크탑 장치용 서프로 더 높은 수준의 보안을 시도하십시오"
```

File: src/ru.json [hunk 4]
```diff
@@ -965,6 +965,17 @@
...
         "rate": "Оценить Surf"
     },
+    "promoDownload": {
+        "mobile": {
+            "appStore": "App Store",
+            "googlePlay": "Google Play",
+            "notice": "Установить приложение"
+        },
+        "desktop": {
+            "download": "Download now for free",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
            "download": "Скачать бесплатно",
```

File: src/ru.json (hunk 2 above)
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
            "notice": "Переходите на новый уровень безопасности. Установите Surf на свой компьютер"
```

File: src/de.json [hunk 5]
```diff
@@ -965,6 +965,17 @@
...
         "rate": "Surf bewerten"
     },
+    "promoDownload": {
+        "mobile": {
+            "appStore": "App Store",
+            "googlePlay": "Google Play",
+            "notice": "Download application"
+        },
+        "desktop": {
+            "download": "Download now for free",
```
Comment (AnnStepanova) [inline src/de.json]:
```suggestion
            "download": "Download desktop app",
```

File: src/de.json [hunk 6]
```diff
@@ -965,6 +965,17 @@
...
+            "notice": "Try a new level of security with Surf for desktop devices"
```
Comment (AnnStepanova) [inline src/de.json]:
```suggestion
            "notice": "Try a new level of security with Surf"
```

File: src/ru.json [hunk 7]
```diff
@@ -965,6 +965,17 @@
...
         "rate": "Оценить Surf"
     },
+    "promoDownload": {
+        "mobile": {
+            "appStore": "App Store",
+            "googlePlay": "Google Play",
+            "notice": "Установить приложение"
+        },
+        "desktop": {
+            "download": "Скачать бесплатно",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
            "download": "Скачать десктоп-приложение",
```

File: src/ru.json [hunk 8]
```diff
@@ -965,6 +965,17 @@
...
+            "notice": "Переходите на новый уровень безопасности. Установите Surf на свой компьютер"
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
            "notice": "Surf выходит на новый уровень безопасности"
```

File: src/ko.json [hunk 9]
```diff
@@ -976,6 +976,17 @@
...
     },
+    "promoDownload": {
+        "mobile": {
+            "appStore": "App Store",
+            "googlePlay": "Google Play",
+            "notice": "애플리케이션 다운로드"
+        },
+        "desktop": {
+            "download": "무료 다운로드하세요",
+            "notice": "데스크탑 장치용 서프로 더 높은 수준의 보안을 시도하십시오"
```
Comment (sertony) [inline src/ko.json]: It happened to be that the text has changed a little:
```
            "download": "Download desktop app",
            "notice": "Try a new level of security with Surf"
```

@kkkhesha Can you help once again?)

**Pull request merged**